assets/prototype/domain/eventEditor/datetimes/dateForm/DateForm.js

```diff
@@ -61,7 +61,7 @@ const formatSecondaryField = (ticketPrice, toString = false) => {
 
 const DateForm = ({ formReset, relatedTickets, tickets = [], title }) => {
 	const { id } = useContext(DateTimeContext);
-	const { description = '', name = '' } = useDatetimeItem({ id });
+	const { description = '', name = '' } = useDatetimeItem({ id }) || {};
```

Comment:
:crying_cat_face:

Reply:
I myself am not OK with that, but it's a temporary fix and needs improvement but may be in a separate PR.

Reply:
Yes, I understood that it might not be smth trivial, that's why I approved the PR.